aioshumway/__init__.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Wrapper around shumway (https://github.com/spotify/shumway) that
+uses asyncio features to make all IO operations non-blocking and compatible
+with the async/await syntax.
+"""
+
+__author__ = 'Matt Obarzanek'
+__version__ = '1.0.0'
+__license__ = 'Apache 2.0'
+__email__ = 'matto@spotify.com'
+__description__ = 'Asynchronous metrics library for ffwd'
+__uri__ = 'https://github.com/spotify/aioshumway'
+
+import asyncio
+import json
+import logging
+
+import shumway
+
+
+FFWD_IP = shumway.FFWD_IP
+FFWD_PORT = shumway.FFWD_PORT
+GIGA_UNIT = shumway.GIGA_UNIT
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.NullHandler())
```

Comment:
Added this going off of: http://docs.python-guide.org/en/latest/writing/logging/#logging-in-a-library